src/resources/postcss/rsvp/_attendee-registration.pcss

```diff
@@ -129,14 +137,30 @@
 		@media(--viewport-medium) {
 			@mixin visually-show;
 		}
+
+		&:hover,
+		&:focus {
+			color: var(--color-text-primary);
+		}
```

Comment:
I just looked at the markup. Shouldn't the hover and focus be on the button instead?

Reply:
Nice! yeah, I think so. Also switched from `disabled` to `inactive` to avoid confusion.